packages/prisma/schema.prisma

```diff
@@ -342,6 +342,7 @@ model EventTypeCustomInput {
   label       String
   type        EventTypeCustomInputType
   required    Boolean
+  hidden      Boolean                  @default(false)
```

Comment:
Why adding a new property instead of adding a new `EventTypeCustomInputType` ? AFAIK inputs of type "hidden" are strings anyways so I don't see much point of having "bool" hidden for example.

Reply:
well EventTypeCustomInputType is for the type of input ( text, number ). what peer mentioned here https://calendso.slack.com/archives/C01NXDSRY9W/p1661256038898709?thread_ts=1661217673.551779&cid=C01NXDSRY9W is to have an additional checkbox to mark the additional custom as input very similar to required checkbox. 

i don't understand how would adding new EventTypeCustomInputType help   ?

Reply:
Hidden should not be a property of a field but a field type on it's own and render them as:

`<input type="hidden" />`

Instead of 

`<input type="text/number/checkbox/etc..." className="hidden" />`

Reply:
i am rendering it as a field type `<input  type={input.hidden ? "hidden" : "number"} />`.  hidden is stored in db so that we know which custom input type(number, text) was marked hidden by checkbox .  

so what i should do now is create new EventTypeCustomInputType of type hidden and not create any checkbox for marking hidden?